modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/IndexStorageImpl.java

```diff
@@ -251,26 +263,26 @@ private String maskCacheIndexName(Integer cacheId, String idxName, int segment)
          * @param reuseList Reuse list.
          * @param innerIos Inner IOs.
          * @param leafIos Leaf IOs.
-         * @param failureProcessor if the tree is corrupted.
          * @throws IgniteCheckedException If failed.
          */
         private MetaTree(
-            final int cacheId,
-            final int allocPartId,
-            final byte allocSpace,
-            final PageMemory pageMem,
-            final IgniteWriteAheadLogManager wal,
-            final AtomicLong globalRmvId,
-            final long metaPageId,
-            final ReuseList reuseList,
-            final IOVersions<? extends BPlusInnerIO<IndexItem>> innerIos,
-            final IOVersions<? extends BPlusLeafIO<IndexItem>> leafIos,
-            final boolean initNew,
+            String treeName,
+            int cacheId,
+            int allocPartId,
+            byte allocSpace,
+            PageMemory pageMem,
+            IgniteWriteAheadLogManager wal,
+            AtomicLong globalRmvId,
+            long metaPageId,
+            ReuseList reuseList,
+            IOVersions<? extends BPlusInnerIO<IndexItem>> innerIos,
+            IOVersions<? extends BPlusLeafIO<IndexItem>> leafIos,
+            boolean initNew,
             @Nullable FailureProcessor failureProcessor,
-            @Nullable PageLockListener lockLsnr
+            PageLockTrackerManager pageLockTrackerManager
         ) throws IgniteCheckedException {
             super(
-                treeName("meta", "Meta"),
+                treeName,
```

Comment:
what was this for?

Reply:
what was what for?

Reply:
Sorry, are there any clues about that original line? What was its purpose?

Reply:
it was a "name for debugging purposes", e.g. it was printed in `toString()`. A name will still be printed, it will just be a little different (but very similar)